src/InstrumentationEngine.Api/InstrumentationEngine.h

```diff
@@ -7433,7 +7433,7 @@ EXTERN_C const IID IID_IInstructionFactory;
         
         virtual HRESULT STDMETHODCALLTYPE DecodeInstructionByteStream( 
             /* [in] */ DWORD cbMethod,
-            /* [in] */ __RPC__in LPCBYTE instructionBytes,
+            /* [size_is][in] */ __RPC__in_ecount_full(cbMethod) LPCBYTE instructionBytes,
```

Comment:
>__RPC__in_ecount_full(cbMethod) [](start = 32, length = 31)

I don't see a corresponding change to the idl.

Reply:
I've updated the idl file